test/mpi/configure.ac

```diff
@@ -512,21 +512,6 @@ AC_SUBST(comm_overlap)
 
 # The multi-thread tests can be disabled by --disable-threads
 threadsdir="threads"
-if test "$enable_threads" != "no" ; then
-    AC_MSG_CHECKING([whether MPI_THREAD_MULTIPLE is supported])
-    AC_RUN_IFELSE([
-            AC_LANG_PROGRAM([#include "mpi.h"],[
-                int provided;
-                MPI_Init_thread(0,0, MPI_THREAD_MULTIPLE, &provided);
-                MPI_Finalize();
-                if (provided != MPI_THREAD_MULTIPLE) {
-                    exit(1);
```

Comment:
I think `return 1` will have the same effect.

Reply:
It does, but there are further complications with the publish test when run as a singleton:
pmi-1
```
Abort(402268449) on node 0 (rank 0 in comm 0): Fatal error in internal_Publish_name: Invalid service name (see MPI_Publish_name), error stack:
internal_Publish_name(72253): MPI_Publish_name(service_name=servname, MPI_INFO_NULL, port_name=otherhost:122) failed
MPIR_pmi_publish(1081)......: Lookup failed for service name servname
```
pmi-2
```
[unset]: PMIU_write error; fd=-1 buf=:    14cmd=job-getid;:
system msg for write_line failure : Bad file descriptor
[2]    85505 segmentation fault  ./a.out
```
pmix (singleton is broken, i have another patch to fix)
```
[2]    28193 segmentation fault  ./a.out
```